mjpeg-view/src/main/java/com/github/niqdev/mjpeg/MjpegViewDefault.java

```diff
@@ -161,7 +163,13 @@ void _surfaceDestroyed(SurfaceHolder holder) {
         }
     }
 
-    void _frameCaptured(Bitmap bitmap) {
+    void _frameCapturedWithByteData(byte[] imageByte,byte[] header) {
```

Comment:
function with a leading `_`you can love it or not. I don't love it

Reply:
I don't love it too, but I just follow other methods format in this class

Reply:
i don't like it either, but i wouldn't be too picky on this